tests/backstop_data/gulp-tasks/data-setup-steps/activity.service.js

```diff
@@ -22,32 +24,24 @@ function setupData () {
 /**
  * @param {number} numberOfActivities number of activities
  * @param {object} params params
- * @returns {Array} list of activities
+ * @returns {Array} list of activity ids
  */
 function createActivities (numberOfActivities, params) {
-  var createUniqueActivity = createUniqueRecordFactory('Activity', ['subject']);
-
-  var activityIds = [];
-
-  for (var i = 0; i < numberOfActivities; i++) {
-    var finalParams = _.extend({
-      source_contact_id: contactsService.activeContact.id,
-      subject: params.activity_type_id + ' ' + (i === 0 ? '' : (i + 1)),
-      activity_date_time: moment().startOf('month').format('YYYY-MM-DD')
-    }, params);
-
-    activityIds.push(createUniqueActivity(finalParams).id);
-  }
-
-  return activityIds;
+  var finalParams = _.extend({
+    source_contact_id: contactService.activeContact.id,
+    activity_date_time: moment().startOf('month').format('YYYY-MM-DD')
+  }, params);
```

Comment:
const :P

You don't need to change this, but instead of using extend you could have used the spread operator here as well.

Or even have something like

```js
const defaultParams = {
  source_contact_id: contactService.activeContact.id,
  activity_date_time: moment().startOf('month').format('YYYY-MM-DD')
};

return _.range(numberOfActivities).map((i) => createUniqueActivity({
  ...defaultParams,
  ....params,
  subject: params.activity_type_id + ' ' + (i === 0 ? '' : (i + 1))
}).id);
```